scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -57,34 +56,49 @@ def compute_measures(filename_tuple):
     nbr_streamlines = len(sft)
     if not nbr_streamlines:
         logging.warning('{} is empty'.format(filename_tuple[0]))
-        return dict(zip(['volume', 'volume_endpoints', 'streamlines_count',
+        return dict(zip(['volume', 'volume_endpoints', 'streamlines_count', 'total_length',
                          'avg_length', 'std_length', 'min_length', 'max_length',
-                         'mean_curvature'], [0, 0, 0, 0, 0, 0, 0, 0]))
+                         'span', 'curl', 'diameter', 'elongation', 'mean_curvature'],
+                        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]))
 
     length_list = list(length(list(sft.streamlines)))
     length_avg = float(np.average(length_list))
     length_std = float(np.std(length_list))
     length_min = float(np.min(length_list))
     length_max = float(np.max(length_list))
+    length_sum = float(np.sum(length_list))
```

Comment:
I would remove the sum, it is too dependent on streamlines count to be interpretatble.